### ISSUE TITLE
Implement account creation

### ISSUE THREAD

**Issue by SlnPons** [massalabs/thyra-plugin-wallet#81]
**Context**
Part of mainnet plan - [Wallet product milestone 2](https://www.notion.so/massa-innoteam/Wallet-85ec39b64f72481f98a0ff31c8a5608e?pvs=4)

--> the objective of this ticket is to add the possibility to create an account for end-users: 
"Massa basic" - entre nickname + password 

**User flow**

* Wallet final designs: https://www.figma.com/proto/LSaOibVDFb1reF6FePVn5l/Massastation?node-id=562-9397&scaling=min-zoom&page-id=324%3A5834&starting-point-node-id=562%3A9449

**How to**

big picture:
- [x] Create associated front-end screens

**Technical details**
*Give the technical insights so anyone in the team can tackle the tasks - Dev*

- [x] set up the react front end #110 
- [x] use the tailwind preset: https://github.com/massalabs/Ui-Kit/issues/8

In another PR: Implement account creation

- [x] #141
- [x] #162
- [x] #140
- [x] #200
- [x] #144
- [x] #209
- [x] #173
- [x] #211
- [x] #179
- [x] Connect the account creation step 2 front-end to back-end /!\ careful there are some rules about nickname and errors messages to update. Please ping PM to align and define.
- [x] #198
- [x] Connect the account creation step 3 front-end to the back-end
- [ ] #166

**Side note**:
- vanilla JS pages will stay, we will add new features on the flow and remove vanilla JS once everything is migrated to React
Existing front-end feature list:
- import
- create
- list my accounts with balance

**QA testing**
User must be able to create an account

**Comment by SlnPons** [comment]
@Thykof can you specify why this ticket is blocked please?

**Comment by Thykof** [comment]
blocked by some stuff to setup like ui-kit (being able to import it, add some component, share the theme...)

**Comment by SlnPons** [comment]
Let's update this ticket @Thykof becaue designs are now ready ;) 
ANd if that's ok for you, let's remove the UX screens. Only use designs to break down your work.

**Comment by Thykof** [comment]
The issue #166 contains some task about delete and transfer features. So This issue is closed.

**Comment by SlnPons** [comment]
Hey @maxime-carabina can you tell me how long the wails pop-up stays open before closing automatically at the end of the process please? 
Ping me if you don't see what I mean ;) 

I've got the feeling that it's not always the same timing and/or that sometimes it's too fast to read the text.

**Comment by maxime-carabina** [comment]
> Hey @maxime-carabina can you tell me how long the wails pop-up stays open before closing automatically at the end of the process please? Ping me if you don't see what I mean ;)
> 
> I've got the feeling that it's not always the same timing and/or that sometimes it's too fast to read the text.

Hello @SlnPons , to give you more details :
- **Delete account** screen in wails is in code review (Nathan made the PR)
- **Sign a transaction** screen in wails is the only screen not yet made. I can develop the screen but the backend isn't ready yet.
  - What do you suggest?

**Comment by SlnPons** [comment]
@Thykof I'm surprised to see that the back-end is not ready yet for the sign transaction as it already exists in Fyne. Can you make sure to sync with Paul & Pierre if needed regarding this point so everyone in the team has the correct info please?

But as discussed, @maxime-carabina my question wasn't linked remaining work. But rather the user flow, at the end of the wails user flow the pop-up is automatically closed after some seconds. What is the current defined time after which the wails windows is closed? --> I think it's too short, we can't read.

**Comment by Thykof** [comment]
Failure screen automatically close after 2 sec, but not in every case. We would need a more precise specification on that.

Success screen automatically close after 2 sec.

The sign transaction feature is done e2e, remaining task is updating wording.
